### DISCUSSION TITLE
QGIS Crashing When Saving Qfield Project (.qgs) – Anyone Experienced This?

### DISCUSSION THREAD

**Discussion by SIG-CCBE** [opengisch/QField#5747] [Bugs]
When I try to save my .qgs project for QField, the project saves successfully, but QGIS freezes (it never finishes, and QGIS becomes unresponsive). I have to force quit every time. 

However, the .qgs file is saved correctly, and after reopening QGIS, I can transfer my project to QFieldCloud using QFieldSync.

Has anyone encountered this issue?

Any ideas on how to fix this?

**Comment by beanzmo** [comment]
Hi @SIG-CCBE  - Does this happen with every project or just with the particular one ? Can you share the project here as a zip file
Best B

**Reply by SIG-CCBE** [reply]
Hi thanks for your reply.
Yes, this happens with all my projects but only when I am logged into the Qfieldsync plugin. 
Otherwise, there is no problem. 
Any idea why ?

**Reply by beanzmo** [reply]
Hi @SIG-CCBE 
Which version are you using of 
QGIS
QFIELD Sync

**Reply by Qgisrookie1** [reply]
It happens to me as well.
Im using:
- QGIS 3.40.3 
- QFIELD Sync 4.14.1

**Reply by SIG-CCBE** [reply]
Hi,
The "solution" I found is to log out of the QFieldSync plugin before saving the .qgs project - no more freezing issues.
Then, sign back in and transfer the project to QFieldCloud.

**Reply by Qgisrookie1** [reply]
Thank you. 
Unfortunately it doesn't seem to be working for me :-/

**Reply by beanzmo** [reply]
Hi - Just to let you know. We have moved all the discussions to our community's  platform. https://community.qfield.org/ 

Maybe you can raise the issue there again and get more support or have the community join in ?
